cesium/features/cadence_features.py

```diff
@@ -7,7 +7,8 @@
 
 
 def double_to_single_step(cads):
-    """Ratios (t[i+2] - t[i]) / (t[i+2] - t[i+1])."""
+    """Ratios t[i+2] - t[i] / (t[i+2] - t[i+1])."""
```

Comment:
This function actually was implemented incorrectly; if you rebase and just accept my changes from 808e0c7a4a63aa55164a7eb873c7a5e718283066 then you shouldn't need to touch this function.